src/components/AppGuides/AppGuide/AppGuide.scss

```diff
@@ -3,9 +3,18 @@
 .breadcrumbContainer {
     margin: 25px 0;
 }
+.appGuideTable {
+	margin: 0 5px 0 5px;
+	width: 95%;
+}
+.appGuideTable>tbody>tr>td, .appGuideTable>thead>tr>th,
```

Comment:
It is, but I think this is cleaner the way the code is written. I'll have to copy/paste the class name into 5-6 places.